### PR TITLE
Alusine-homework week1

### PR DESCRIPTION
**Pull request by alquinex**

### CODE REVIEW

File: week1/main.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+// This is a class based component that has two props that are reuseable
+// as demonstrated in lines 47 through 49.
+class ListItems extends React.Component {
```
Comment (ArcoMul) [inline week1/main.js]: `ListItem` would be a better name I think, since it's only one item of the list.

File: week1/main.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    );
+  }
+}
+
+// This component serves as parent component which renders the component
+// being nested in it.
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```
Comment (ArcoMul) [inline week1/main.js]: Good that you use state! Exactly what it is meant for.

File: week1/main.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+          done: false,
+        },
+      ],
+    };
+  }
+  render() {
+    return (
+      <div className="list-items">
+        <h1 className="title">Static List</h1>
+        <ListItems description="Get out of bed" deadline="2017-09-13" />
```
Comment (ArcoMul) [inline week1/main.js]: Please make a container component which holds the static list, and the same for the dynamic list. Two components called `StaticList` and `DynamicList` for example. Pass the right data to these container components and renders the `ListItem`s.

File: week1/main.js (hunk 1 above)
Comment (alquinex) [reply week1/main.js]: Hi @ArcoMul,
Thanks for your time and for the feedback. The changes have been made.

**Comment by ArcoMul** [comment]
Hi @alquinex, the structure of your components is not very clear. You now have a component called StaticList, but it doesn't contain a list, it renders a list item. Your DynamicList uses a component called StaticList. A bit confusing.

Try to make your component structure like this:
```jsx
<div className="lists">
        <h1 className="title">Static List</h1>
        <StaticList /> { /* Renders your static list, using the same component for a list item as DynamicList */ }
        <h1 className="title">Dynamic List</h1>
        <DynamicList /> { /* Renders your dynamic list, using the same component for a list item as StaticList */ }
</div>
```

It's a bit hard to explain / demonstrate. See if you understand what I am saying. If not, talk to me on Slack then I can explain it more thoroughly, or we can make a call for a few minutes.

**Comment by alquinex** [comment]
Hi @ArcoMul,
The changes have been made. Thanks for your guide -:)

**Pull request closed without merging**